wrappers/java/wrapper

```diff
@@ -33,23 +33,27 @@ wrapper_dir=$(dirname "$0")
 cd "$wrapper_dir"
 
 if [ "$action" == "version" ]; then
-	exec ant -S -q version
+	exec mvn --version
```

Comment:
It's not maven's version that needs to be printed here, but the project's version. See my comment there was an example command-line